### PR TITLE
Automatically shut wells with no connections

### PR DESCRIPTION
**Pull request by joakim-hove**
This PR will automatically SHUT a well with zero connections in the input layer. Lots of nicely formatted warnings all over the place

### CODE REVIEW

**Comment by joakim-hove** [comment]
jenkins build this please

**Comment by joakim-hove** [comment]
jenkins build this please

**Comment by totto82** [comment]
Thanks. I will check it out!

**Comment by totto82** [comment]
It works as announced!

**Comment by joakim-hove** [comment]
jenkins build this please

File: src/opm/parser/eclipse/EclipseState/Schedule/KeywordHandlers.cpp [hunk 1]
```diff
@@ -153,11 +153,19 @@ namespace {
...
                 auto well2 = std::shared_ptr<Well>(new Well( this->getWell(name, handlerContext.currentStep)));
                 auto connections = std::shared_ptr<WellConnections>( new WellConnections( well2->getConnections()));
-                connections->loadCOMPDAT(record, handlerContext.grid, handlerContext.fieldPropsManager);
+                connections->loadCOMPDAT(record, handlerContext.grid, handlerContext.fieldPropsManager, handlerContext.keyword.location());
                 if (well2->updateConnections(connections, handlerContext.grid, handlerContext.fieldPropsManager.get_int("PVTNUM"))) {
                     this->updateWell(std::move(well2), handlerContext.currentStep);
                     wells.insert( name );
                 }
+
+                if (connections->size() == 0 && well2->getConnections().size() == 0) {
```
Comment (bska) [inline src/opm/parser/eclipse/EclipseState/Schedule/KeywordHandlers.cpp:162]: Any reason not to use `WellConnections::empty()` here?

**Review by bska — COMMENTED** [review]
Looks good for the most part.  Just one remark about using `WellConnections::empty()` versus `WellConnections::size() == 0` in one condition.

File: src/opm/parser/eclipse/EclipseState/Schedule/KeywordHandlers.cpp (hunk 1 above)
Comment (joakim-hove) [reply src/opm/parser/eclipse/EclipseState/Schedule/KeywordHandlers.cpp:162]: No reason - I'll update

**Comment by joakim-hove** [comment]
jenkins build this please

**Pull request merged**